src/fragments/lib/auth/ios/advanced/advanced.mdx

```diff
@@ -0,0 +1,87 @@
+## Subscribing Events
+
+You can take specific actions when users sign-in or sign-out by subscribing authentication events in your app. Please see our [Hub Module Developer Guide](/lib/utilities/hub) for more information.
+
+## Identity Pool Federation
+
+You can alternatively use `federateToIdentityPool` to get AWS credentials directly from Cognito Federated Identities and not use User Pool federation. If you have logged in with `Auth.signIn` you **can not** call `federateToIdentityPool` as Amplify will perform this federation automatically for you in the background. In general, you should only call `Auth.federatedSignIn()` when using OAuth flows. 
```

Comment:
can you please explain the use case here? rather than the technical implementation details? Why would a customer want to use Identity pool federation?